server/index.js

```diff
@@ -18,6 +18,7 @@ client.connect();
 
 const express = require('express');
 const app = express();
+const db = require('./queries');
```

Comment:
I dig putting all the db interaction stuff in a separate place from the routing logic 👍 the calls look clean